### PR TITLE
Creating a data catalog with Obsidian and Dataview

### LINKED ISSUE #20: Tutorial: "How to create a data catalog out of markdown files using free tools?"
## Acceptance
- [ ] write a tutorial with the following content structure

image

### PR DESCRIPTION
**Pull request by olayway**
Closes #20 

Additionally at the beginning of the written tutorial, there is the following gif added:

dataview

### CODE REVIEW

**Comment by rufuspollock** [comment]
@demenech can you review.

**Comment by demenech** [comment]
> @demenech can you review.

yes. Will do ASAP.

File: examples/obsidian-dataview/README.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+
+## Summary
+
+And there you have it! We've walked through the process of creating a personal data catalog using markdown files, the Obsidian application, and its Dataview plugin. We've transformed scattered data into a well-organized, easy-to-navigate, and visually pleasing catalog. Whether you've used our example of a Harry Potter character catalog or applied these steps to a different topic of your choosing, we hope you've found this tutorial helpful and empowering.
+
+Happy data cataloging!
+
+---
+
+You can find the vault created in this tutorial [here](https://github.com/datopian/markdowndb/tree/main/examples/dataview).
```
Comment (demenech) [inline examples/obsidian-dataview/README.md:192]: I think this link is going to be broken even after we merge

**Review by demenech — COMMENTED** [review]
Wow. Super well-written and engaging tutorial. I added just a comment to double-check something before merging.

File: examples/obsidian-dataview/README.md (hunk 1 above)
Comment (rufuspollock) [reply examples/obsidian-dataview/README.md:192]: I'm going to fix this now.

**Pull request merged**